css/main.css

```diff
@@ -75,22 +75,26 @@ body > * {
 
 .navbar-brand > img {
 	max-height: 22px;
-	opacity: 0.5;
+	opacity: 0.7;
 }
 
 .navbar-brand:hover > img {
 	opacity: 1;
 }
 
+.nav-link {
+	color: rgba(255, 255, 255, .7)
+}
+
 /** Footer formatting **/
 .footer {
-	background-color: #000e33;
+	background-color: #003368;
 	color: #ffffff;
 	clear: both;
 	margin: 1em -30px -30px -30px;
 	font-size: 1.25em;
 	flex-grow: 1;
-	box-shadow: inset 0px 10px 8px 0px #00071a;
+	/* box-shadow: inset 0px 10px 8px 0px #00071a; */
```

Comment:
If this line is commented, is there a reason we shouldn't just remove it?

Reply:
agree; forgot to delete the line. good catch